platform/iphone/godot_iphone.cpp

```diff
@@ -38,11 +38,8 @@
 
 static OSIPhone *os = NULL;
 
-extern "C" {
```

Comment:
I'm an Objective-C guy, so again any clarity on the thinking behind here would be helpful.